lingpy/basic/parser.py

```diff
@@ -364,7 +365,9 @@ def _apply(key, s, *args, **kwargs):
             try:
                 res = function(s, *args, **kwargs)
             except:
-                raise ValueError('Could not convert item "{0}" (ID: {1}).'.format(s, key))
+                print(s, key)
```

Comment:
btw: using the debug functionality would be much better in many cases. What we need, for the users (me included), however, is one short howto that shows how one can switch between the log-levels in an interactive session. I know it's possible, as I have been doing that, but having some easy way to switch, or at least a howto is something I could also offer users who complain that their files don't really load.